docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,1595 @@
+    - [Timing Optimization Using repair_timing](#Timing-Optimization-Using-repair_timing)
+    - [Timing Optimization Based On Multiple Corners](#Timing-Optimization-Based-On-Multiple-Corners)
+    - [Fixing Setup Violations](#Fixing-Setup-Violations)
+    - [Fixing Hold Violations](#Fixing-Hold-Violations)
+  - [Clock Tree Synthesis](#Clock-Tree-Synthesis)
+    - [Reporting Clock Skews](#Reporting-Clock-Skews)
+    - [Reporting CTS Metrics](#Reporting-CTS-Metrics)
+
+## Introduction
+
+This document describes a tutorial to run the complete OpenROAD based
+flow from RTL-to-GDS using
+[OpenROAD Flow Scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts).
+This tutorial also includes examples of useful design explorations and
+manual usage in key flow stages to help user gain a good understanding of
+the [OpenROAD](https://openroad.readthedocs.io/en/latest/main/README.html)
+application flow, data organization, GUI and commands.
+
+This is intended for:
+
+-   Beginners or new users with some understanding of basic VLSI
+    design flow. User will learn the basics of installation to use
+    [OpenROAD-flow-scripts](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html)
+    for the complete RTL-to-GDS flow.
+-   Users already familiar with the OpenROAD application and flow but
+    would like to learn more about specific features and commands.
+
+## User Guide
+
+-   This tutorial requires a specific directory structure built by
+    OpenROAD-flow-scripts (ORFS). Do not modify this structure or
+    underlying files since this will cause problems in the flow execution.
+-   User can run the complete RTL-to-GDS flow and learn specific flow
+    sections independently. This allows user to learn the flow and tool
+    capabilities at your own pace, time and preference.
+-   Results shown such as images or outputs of reports and logs could
+    vary based on release updates. However, the main flow and command
+    structure should generally apply.
+
+Note: Please submit any problem found under Issues in the GitHub repository
+[here](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/issues)
+
+## Getting Started
+
+This section describes the environment setup to build ORFS and get ready
+to execute the RTL-to-GDS flow of the open-source design `ibex` using the
+`sky30hd` technology.
+
+`ibex` is a 32 bit RISC-V CPU core (`RV32IMC/EMC`) with a two-stage
+pipeline.
+
+### Setting Up The Environment
+
+Use the `bash` shell to run commands and scripts.
+
+To install OpenROAD Flow Scripts refer to the [Getting Started with
+OpenROAD Flow](../user/GettingStarted.md) documentation.
+
+```
+git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
+cd OpenROAD-flow-scripts
+```
+
+If ORFS is already installed but needs updating then use use the
+following commands:
+
+```
+git checkout master
+git pull
+```

```

Comment:
It's best to just remove this section and keep info in one place. The second script is missing `git submodule update`.

Reply:
Thanks @rovinski  as we're already pointing installation links, removing it `git clone`